model/task/db.go

```diff
@@ -160,6 +160,32 @@ var (
 		},
 	}
 
+	sortDisplayName = bson.M{
+		"$sort": bson.M{
+			"display_name": 1,
+		},
+	}
+
+	project = bson.M{
```

Comment:
I think this needs a better name  maybe `projectBuildVariant`? When i read the usage below i was confused about what this did. Especially since this is so far away from its usage.

Reply:
Possibly even move this down so it is colocated with where it's being used.